README.md

```diff
@@ -23,9 +23,9 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 BH-Observer Portfolio <a name="about-project"></a>
+# 📖 To-D0 List <a name="about-project"></a>
 
-**Portfolio** is a website that is used to showcase the projects and abilities.
+**To-do list** is a website that is used to showcase the projects and abilities.
 
 ## 🛠 Built With <a name="built-with"></a>
```

Comment:
[OPTIONAL] The live demo link has a URL that is not relevant to this project. Kindly update the link with a URL for this project and if it is not required for you to deploy this project then you can delete the whole live demo section.  

Otherwise, well done with the passing tests. 👏🏾🚀 

![image](https://github.com/BHObserver/To-Do-List/assets/38088818/69754a55-f58b-4ccd-bd50-2df14c55ee63)